designs/oauth/oauth.md

```diff
@@ -0,0 +1,26 @@
+# OAuth Flow
+
+Updates to the create a connection flow to account of OAuth feature for Twitter and Salesforce.
+
+![Image of choosing connection](img/chooseconnector.png)
+
+In this case, user chooses Twitter Connector.
+
+![Image of choosing object](img/connecttwitter.png)
+
+User clicks on "Connect Twitter" button to start OAuth flow.
+
+![Image of choosing object](img/oauth.png)
+
+1. Clicking on button opens a new tab and takes user there.
```

Comment:
Are we sure that the we want to open a new tab. From what I see being usually used is that user is redirected to the service provider in the same tab. At the end of the OAuth flow user is redirected back to Syndesis, at this point if we close that tab then the information on OAuth outcome would be lost, and if we keep it open and let the user continue from there the original tab would contain outdated information or would need to be closed.